src/payment/strategies/paypal-commerce/paypal-commerce-funding-key-resolver.ts

```diff
@@ -0,0 +1,32 @@
+export default function paypalCommerceFundingKeyResolver(methodId: string, gatewayId?: string) {
```

Comment:
Can you make this a class and inject it into `PaypalCommercePaymentStrategy` so you can test it in isolation?

Reply:
Okay